doc/classes/TextEdit.xml

```diff
@@ -673,15 +673,15 @@
 		<method name="get_visible_line_count" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the number of visible lines, including wrapped text.
+				Returns the maximum number of lines that can visually fit in the control, rounded down.
```

Comment:
Maybe somethings like "Returns the number of lines that can visually fit, rounded down, based on this control's height."
Since it only really uses the height, not the whole rect.

Reply:
That's good